tests/test_examples.py

```diff
@@ -24,7 +24,15 @@ def test_sqrt():
 def test_mnist():
     skip_if_not_available(modules=['bokeh'])
     with tempfile.NamedTemporaryFile() as f:
-        mnist_test(f.name, 1)
+        mnist_test(f.name, 1, True)
```

Comment:
Probably just run this test once with Bokeh, it'll just slow down the test suite to run it twice, and we don't really gain much from it.